source/lib/all-events.ts

```diff
@@ -1,40 +1,120 @@
 import {readFile} from 'node:fs/promises';
+import type {EventDirectory, EventDirectoryEvent} from './types.ts';
 
-async function getAll(): Promise<string[]> {
-	const data = await readFile('eventfiles/all.txt', 'utf8');
-	const list = data.split('\n').filter(element => element !== '');
-	return list;
+async function getAll(): Promise<EventDirectory[]> {
+	const data = await readFile('eventfiles/directory.json', 'utf8');
+	return JSON.parse(data) as EventDirectory[];
 }
 
 export async function count(): Promise<number> {
-	const allEvents = await getAll();
+	const allEvents = (await find('', [], []));
```

Comment:
Die Klammern müssten hier überflüssig sein, oder? Oder hat xo da irgendwas gegen?

```suggestion
	const allEvents = await find('', [], []);
```